### PR TITLE
Add clause about doxing

### LINKED ISSUE #53: Doxing
I think publishing someone else's contact information should be explicitly called out in the harassment clause.

### PR DESCRIPTION
**Pull request by unixorn**
I'm not entirely happy with the phrasing, but making a PR to make discussion easier.

Closes #53.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ As contributors and maintainers of this project, we pledge to respect all people
 
 We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
 
-Examples of unacceptable behavior by participants include the use of sexual language or imagery, derogatory comments or personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
+Examples of unacceptable behavior by participants include the use of sexual language or imagery, derogatory comments or personal attacks, trolling, public or private harassment including but not limited to publishing other people's private information or electronic or physical address, insults, or other unprofessional conduct.
```
Comment (kerrizor) [inline CODE_OF_CONDUCT.md]: Sadly, the list is getting a little unwieldy at this length. How about we bullet it?

```markdown
Examples of unacceptable behavior by participants include:

+ the use of sexual language or imagery
+ derogatory comments or personal attacks
+ behavior intended only to disrupt (trolling)
+ public or private harassment
+ dissemination of other people's private information or electronic or physical address
+ insults or other unprofessional conduct
```

Comment (unixorn) [inline CODE_OF_CONDUCT.md]: I like that better. I'll update the PR accordingly.

**Comment by unixorn** [comment]
@kerrizor I agree that the list has gotten too long. I split it into bullets to make it less unwieldy.

**Comment by strand** [comment]
I believe that the language should read "Publishing other's private information, such as physical or electronic addresses." Would you accept a _yo dawg_ PR, that is, if I make a pull request on your pull request would you merge it?

**Comment by unixorn** [comment]
I like your phrasing better, @strand. I'd accept a PR but it was a one line change so I've updated the PR already.

**Comment by strand** [comment]
"Private" may be difficult to define and cause weaseling. Should we remove it, replace it wit an alternative (personal, protected). If we remove it perhaps add "without their permission"? :-/

**Comment by CoralineAda** [comment]
I like “without their permission”.

**Comment by CoralineAda** [comment]
Adding this to 1.2.0.

**Pull request closed without merging**